xinference/model/llm/llm_family.json

```diff
@@ -2070,6 +2070,64 @@
       ]
     }
   },
+  {
+    "version": 1,
+    "context_length": 32768,
+    "model_name": "mistral-8x7b-instruct-v0.1",
```

Comment:
The model name should be `mixtral`. And this model name is too long for SDK and cmdline users. I think `mixtral-instruct-v0.1` is ok.
Same as other places in the document.